Update TSC Attendance Governance (#610)
eea-oasis/baseline: docs/governance/governance.md

No legal entity (or set of entities controlled by a single party) shall hold more than three seats out of eleven (or two seats out of seven or nine) on the TSC during any given period.

The TSC's members are elected annually in September/October, concluding on the last day of that month, with the first elections held in September, 2020. The period for nominations will be announced no less than 30 days prior to elections. The method and management of the nominating process and the elections will be communicated to the community by the various [channels](../../community/open-source-community.md#communications-connecting-with-each-other-directly) in that timeframe.

#### Removal

A TSC member is eligible to lose their seat upon missing two consecutive TSC meetings or three total during a period between elections, or for violating the Code of Conduct.

> If a TSC member does not comply with the attendance policy, a simple majority vote for removal is conducted by Oasis for the remaining TSC members that are not being considered for removal to vote.

Reviewer: Is it necessary for the vote to be "conducted by Oasis"?

> If a TSC member does not comply with the attendance policy, a simple majority vote for removal is conducted by Oasis for the remaining TSC members that are not being considered for removal to vote.

Reviewer 2: ```suggestion
If a TSC member does not comply with the attendance policy, a simple majority vote of the remaining TSC members that are not being considered for removal is sufficient to remove the TSC member.
```
